ui/shared/ComparePageTitle.jsx

```diff
@@ -18,6 +18,20 @@ export default class ComparePageTitle extends React.Component {
     };
   }
 
+  componentDidUpdate(prevProps) {
+    const { pageTitleQueryParam, title } = this.props;
+    if (
+      prevProps.pageTitleQueryParam !== pageTitleQueryParam ||
+      prevProps.title !== title
+    ) {
+      // eslint-disable-next-line react/no-did-update-set-state
```

Comment:
Rather than disabling this rule, can you find a workaround so you don't need to?